addons/binding/org.openhab.binding.hs110/LICENSE

```diff
@@ -0,0 +1,217 @@
+Eclipse Public License, Version 1.0 (EPL-1.0)
```

Comment:
There is mention about it in http://docs.openhab.org/developers/development/guidelines.html#b-osgi-bundles